Bottom: ae736c374725c2c2ef1ee2e01ba44a1dd7026a30
Top:    5217b749fa9f6b49522903b024fefe25e42f242e
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-06-13 14:30:18 +0800

vhost: correctly assign vhost message type

Type of message should not be VHOST_IOTLB_MISS but
VHOST_IOTLB_MSG. Previous code work by chance since their value is
equal.

Signed-off-by: Jason Wang <jasowang@redhat.com>


---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 895eaa2..0493ba0 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -1124,7 +1124,7 @@ static int vhost_iotlb_miss(struct vhost_virtqueue *vq, u64 iova, int access)
 	struct vhost_msg_node *node;
 	struct vhost_iotlb_msg *msg;
 
-	node = vhost_new_msg(vq, VHOST_IOTLB_MISS);
+	node = vhost_new_msg(vq, VHOST_IOTLB_MSG);
 	if (!node)
 		return -ENOMEM;
